package.json

```diff
@@ -39,16 +39,16 @@
     "jest-cli": "^20.0.0",
     "jsdom": "^9.5.0",
     "lodash-webpack-plugin": "^0.11.4",
-    "node-sass": "^4.5.0",
+    "node-sass": "^4.9.0",
```

Comment:
This is a rather complicated package to build. Could we submit this as a separate PR so we can get all the other changes in first?